source/change-streams/change-streams.rst

```diff
@@ -388,26 +388,26 @@ Single Server Topologies
 
 Presently, change streams cannot be initiated on single server topologies as they do not have an oplog.  Drivers MUST NOT throw an exception in this scenario, but instead rely on an error returned from the server.  This allows for the server to seamlessly introduce support for this in the future, without need to make changes in driver code.
 
-startAtClusterTime
+startAtOperationTime
 ^^^^^^^^^^^^^^^^^^
 
 :since: 4.0
 
-``startAtClusterTime`` specifies that a ``changeStream`` will only return changes that occurred at or after the specified ``Timestamp``.
+``startAtOperationTime`` specifies that a ``changeStream`` will only return changes that occurred at or after the specified ``Timestamp``.
 
-The server expects ``startAtClusterTime`` as a bson document of the form
+The server expects ``startAtOperationTime`` as a bson Timestamp
 
 .. code:: typescript
 
     {
       ts: Timestamp;
     }
 
-Drivers MUST allow users to specify a ``startAtClusterTime`` option in the ``Collection.watch`` and ``Database.watch`` helpers. They MUST allow users to specify this value as a raw ``Timestamp``, and MUST NOT force users to wrap the timestamp as the server specifies.
+Drivers MUST allow users to specify a ``startAtOperationTime`` option in the ``Collection.watch`` and ``Database.watch`` helpers. They MUST allow users to specify this value as a raw ``Timestamp``.
 
-``startAtClusterTime`` and ``resumeAfter`` are mutually exclusive; if both ``startAtClusterTime`` and ``resumeAfter`` are set, the server will return an error. Drivers MUST NOT throw a custom error, and MUST defer to the server error.
+``startAtOperationTime`` and ``resumeAfter`` are mutually exclusive; if both ``startAtOperationTime`` and ``resumeAfter`` are set, the server will return an error. Drivers MUST NOT throw a custom error, and MUST defer to the server error.
 
-If neither ``startAtClusterTime`` or ``resumeAfter`` are specified, and the server version is >= ``4.0`` drivers MUST set a default ``startAtClusterTime`` value using a ``$clusterTime`` from any server response (ex: ``ismaster``). This allows change streams to be resumed before the first notification is received.
+If neither ``startAtOperationTime`` or ``resumeAfter`` are specified, and the server version is >= ``4.0`` drivers MUST set a default ``startAtOperationTime`` value using an ``operationTime`` from any server response (ex: ``ismaster``). This allows change streams to be resumed before the first notification is received.
```

Comment:
AFAIK all server responses return an `operationTime`. The server spec for `startAtOperationTime` actually states that the operation time must be obtained as early as possible, rather than on the fly.

I will reword it to explain this.